config/initializers/alaveteli_features.rb

```diff
@@ -12,6 +12,8 @@
   annotations
   alaveteli_pro
   projects
+  pro_batch_access
+  pro_batch_category_ui
```

Comment:
We set these on a per-actor basis – does it make sense to include them here?

I actually think `pro_batch_access` can be done away with entirely. If you're a pro, you should be able to create batches without needing a feature flag set.